server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationSent.ts

```diff
@@ -16,18 +17,29 @@ export default class SufficientInformationSent implements TasklistPage {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   constructor(public readonly body: Record<string, unknown>) {}
 
-  user: ApprovedPremisesUser
+  userName: string
 
-  static async initialize(
-    body: Record<string, unknown>,
-    assessment: Assessment,
-    token: string,
-    dataServices: DataServices,
-  ) {
-    const user = await dataServices.userService.getUserById(token, assessment.application.createdByUserId)
+  emailAddress: string
+
+  phoneNumber: string
+
+  static async initialize(body: Record<string, unknown>, assessment: Assessment) {
```

Comment:
For these TasklistPages we use the `initialize` method for async work. 
Given this is no longer async we can move this functionality to the constructor and remove the `initialize` method